### PR TITLE
Adding new resource type version to manage influxdb Query Language via

### PR DESCRIPTION
**Pull request by Q-Storm**
#### Pull Request (PR) description
Extend the configuration by the resource "version" to configure the Query Language when adding a Data Source "InfluxDB"

#### This Pull Request (PR) fixes the following issues

No issue opened for this.

### CODE REVIEW

File: lib/puppet/type/grafana_datasource.rb [hunk 1]
```diff
@@ -45,6 +45,10 @@
     desc 'The datasource type'
   end
 
+  newproperty(:version) do
+    desc 'The version typo for Influx (optional)'
```
Comment (alexjfisher) [inline lib/puppet/type/grafana_datasource.rb:49]: `typo`? :)

File: lib/puppet/provider/grafana_datasource/grafana.rb [hunk 2]
```diff
@@ -68,6 +68,7 @@ def datasources
           name: datasource['name'],
           url: datasource['url'],
           type: datasource['type'],
+          version: resource[:version],
```
Comment (alexjfisher) [inline lib/puppet/provider/grafana_datasource/grafana.rb:71]: Is this correct? Shouldn't this be returning the datasource as returned by the API?

ie.
```suggestion
          version: datasource[:version],
```
??

File: lib/puppet/type/grafana_datasource.rb (hunk 1 above)
Comment (Q-Storm) [reply lib/puppet/type/grafana_datasource.rb]: you are right, just fixed

File: lib/puppet/provider/grafana_datasource/grafana.rb (hunk 2 above)
Comment (Q-Storm) [reply lib/puppet/provider/grafana_datasource/grafana.rb]: yes this shoud be datasource, I updated the pullrequest.

Comment (Q-Storm) [reply lib/puppet/provider/grafana_datasource/grafana.rb]: I will also provide a test for this, feature

**Pull request closed without merging**